portfolio-jsm/src/components/NavBar.tsx

```diff
@@ -72,19 +72,74 @@ const NavBar = () => {
         </button>
       </div>
       {/* MOBILE */}
-      <div className="mt-5 md:hidden" onClick={toggleMenu}>
-        <Image
-          src={theme === "dark" ? "./menu-dark.svg" : "./menu-light.svg"}
-          alt=""
-          width={24}
-          height={24}
-        />
+      <div className="mt-5 flex flex-row md:hidden">
+        <button onClick={toggleTheme} className="mr-4">
+          {theme === "dark" ? (
+            <Image src="./moon.svg" alt="" width={20} height={20} />
+          ) : (
+            <Image src="./sun.svg" alt="" width={20} height={20} />
+          )}
+        </button>
+        <div onClick={toggleMenu}>
+          <Image
+            src={theme === "dark" ? "./menu-dark.svg" : "./menu-light.svg"}
```

Comment:
Same suggestion as other PR. In fact you're doing what I suggest here in the other Image. Use a ternary for the src instead of entire different elements.